apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -41,13 +41,15 @@ type OpenTelemetryCollectorSpec struct {
 	// MaxReplicas sets an upper bound to the autoscaling feature. If MaxReplicas is set autoscaling is enabled.
 	// +optional
 	MaxReplicas *int32 `json:"maxReplicas,omitempty"`
-
 	// Autoscaler specifies the pod autoscaling configuration to use
 	// for the OpenTelemetryCollector workload.
 	//
 	// +optional
 	Autoscaler *AutoscalerSpec `json:"autoscaler,omitempty"`
-
+	// TargetCPUUtilization sets the target average CPU used across all replicas.
+	// If average CPU exceeds this value, the HPA will scale up. Defaults to 90 percent.
+	// +optional
+	TargetCPUUtilization *int32 `json:"targetCPUUtilization,omitempty"`
```

Comment:
@kevinearls could you please review this PR.

I am curious if this should be moved into `.spec.autoscaler.`

Reply:
> In the future we should put any other metrics in spec.autoscaler too.

We should put this comment somewhere in the CRD or book a ticket.

Reply:
Thanks for pointing this out! I moved `TargetCPUUtilization` to the appropriate place in the spec now.  I also created https://github.com/open-telemetry/opentelemetry-operator/issues/1115 to track moving `MaxReplicas` and `MinReplicas` to `.spec.autoscaler`.  I don't mind taking this issue on in a followup PR.